{"name": "Попытка обновления с невалидными данными", "status": "passed", "steps": [{"name": "1. Создание тестовой задачи", "status": "passed", "start": 1746962021188, "stop": 1746962021983}, {"name": "2. Невалидное обновление: числовое имя", "status": "passed", "start": 1746962021983, "stop": 1746962022258}, {"name": "3. Невалидное обновление: слишком длинное описание", "status": "passed", "start": 1746962022258, "stop": 1746962025936}], "start": 1746962021188, "stop": 1746962025940, "uuid": "d5c37642-bd8a-4694-827c-13aab69d572e", "historyId": "0f433ab283ae13bf858e64624eead8ef", "testCaseId": "0f433ab283ae13bf858e64624eead8ef", "fullName": "tests.api.test_tasks.TestTaskAPINegative#test_update_task_negative", "labels": [{"name": "story", "value": "Негативные тесты"}, {"name": "feature", "value": "Task API CRUD Operations"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTaskAPINegative"}, {"name": "host", "value": "DESKTOP-33T1J6E"}, {"name": "thread", "value": "16396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}